ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -203,6 +203,10 @@ defaults <- node_group <- node_group.node <- node
     uplink_ptp:
       enable: < boolean >
 
+    # Enable MacSec on all uplinks | Optional
+    uplink_macsec:
+      profile: "< MacSec profile name >"
```

Comment:
Would we need a datamodel for defining the macsec profile itself? Or can we just use the `eos_cli_config_gen` one directly?

Reply:
I believe it's fine to define the profile directly with eos_cli_config_gen. This is similar to current `p2p_uplinks_qos_profile: < qos_profile_name >`

Reply:
ok